tensorcircuit/mpscircuit.py

```diff
@@ -405,7 +383,7 @@ def reduce_tensor_dimension(
         tensor_left: Tensor,
         tensor_right: Tensor,
         center_left: bool = True,
-        **rules: Any,
+        split: Dict[str, Any] = {},
```

Comment:
better  = None, since dict is a mutable object?

Reply:
That's true. Previously I thought only list cause this problem